docs/workshop/index.md

```diff
@@ -15,9 +15,9 @@ In this training course, we will provide an introduction to the following topics
 
 3. **Complete the ["Get Started" tutorial](https://prairielearn.readthedocs.io/en/latest/getStarted/)**, which will show you how to create:
```

Comment:
Ah, shoot, it looks like Prettier tries to enforce formatting that isn't compatible with how mkdocs treats Markdown 😢 here's an alternative:

```suggestion
3. **Complete the ["Get Started" tutorial](https://prairielearn.readthedocs.io/en/latest/getStarted/)**, which will show you how to create course instances, assessments, and questions.
```